fluentd/Dockerfile.centos7

```diff
@@ -46,7 +46,8 @@ RUN mkdir -p ${HOME} && mkdir -p /etc/fluent/plugin && \
       fluent-plugin-secure-forward \
      'fluent-plugin-systemd:<0.1.0' \
       fluent-plugin-viaq_data_model \
-     'fluent-plugin-remote-syslog:1.1'
+     'fluent-plugin-remote-syslog:1.1' \
+     'fluent-plugin-prometheus:<1.0.0'
```

Comment:
@richm do we know what the equivalent package name will be for downstream yet?

Reply:
in general, RPM package names for ruby gems are rubygem-`$fluentd_gem_name` - so rubygem-fluent-plugin-prometheus